todo/src/compoents/Item.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { addTodo, toggleTodo, deleteTodo} from '../actions';
+
+
+class Item extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            todoText: ''
+        }
+    }
+
+    handleChanges = ev => {
+        this.setState( {[ev.target.name]: ev.target.value})
+
+    }
+
+    handleNewTodo = ev => {
+        ev.preventDefault();
+        this.props.addTodo(this.state.todoText)
+        this.setState({
+            todoText: ''
+        })
+    }
+
+    handleDelete = ev => {
+        ev.preventDefault();
+        this.props.deleteTodo()
+    }
+
+    
+
+
+    render() {
+        return (
+            <div>
+                {this.props.todos.map((todo, index) => (
+                    <div key={index}>
+                        <h3 onClick={() => this.props.toggleTodo(index)} className={todo.completed ? 'completed' : null}>{todo.value}</h3>
```

Comment:
this is too long, try:
```suggestion
<h3 
    onClick={() => this.props.toggleTodo(index)} 
    className={ todo.completed ? 'completed' : null}
>
    {todo.value}
</h3>
```